acrobat/scripts/scripts.js

```diff
@@ -391,4 +402,7 @@ const { ietf } = getLocale(locales);
       window.dispatchEvent(imsIsReady);
     }
   }, 1000);
+
+  const { default: geoPhoneNumber } = await import('./geo-phoneNumber.js');
+  geoPhoneNumber();
 }());
```

Comment:
let's make sure to test this with the edgeworker. Since we inline this module, I expect we will need to make a corresponding change to replace this relative import with an absolute path.

Reply:
can we / should we defer this network activity to give the page a chance to load?  is it critical to page load?

Reply:
will need to do someting like https://github.com/adobecom/dc/blob/stage/edgeworkers/Acrobat_DC_web_prod/main.js#L131